qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/DependencyManager.groovy

```diff
@@ -76,21 +77,23 @@ class DependencyManager {
         try {
             this.viewModel = new ViewModel()
             viewModel.affiliations.addAll(customerDbGateway.allAffiliations)
```

Comment:
FYI: This Line results in a "java.sql.SQLException" since the backend is not implemented yet and should be commented out for local testing.

Reply:
Works for me... 
The functionality is already present.